examples/bottle/README.md

```diff
@@ -0,0 +1,36 @@
+# Bottle Sample Application for Pybrake
+
+## About the application:
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
+```
+
+2.  Run the localhost server
+
+```bash
+python main.py
+```
+
+3. To retrieve the responses, append the endpoints to the localhost URL with a `/`.
+
+Use the below curl commands to interact with the endpoints. The endpoints require an api-key HTTP header.
+
+```bash
+curl "http://localhost:3000/date/" -H 'api-key: b761be830f7c23ebe1c3250d42c43673' 
```

Comment:
@kyrylo  If I'm not mistaken, this `api-key` should be removed from the request. Am I correct?

Reply:
I think that was our latest idea, but it would be better to verify with @scottsbaldwin.

Reply:
That's correct. We can omit the api-key for the example application.